README.md

```diff
@@ -61,7 +61,7 @@ We introduce the major version into the backup path to smoothen the [major versi
 The new operator configuration can set a compatibility flag *enable_spilo_wal_path_compat* to make Spilo look for wal segments in the current path but also old format paths.
 This comes at potential performance costs and should be disabled after a few days.
 
-The newest Spilo image is: `registry.opensource.zalan.do/acid/spilo-14:2.1-p6`
+The newest Spilo image is: `registry.opensource.zalan.do/acid/spilo-14:2.1-p7`
```

Comment:
@yoshihikoueno please note, that spilo image should be changed in more places (inside code and configs)

```sh
✔ /u01/git/zalando/postgres-operator [master|✔] 
07:23 $ grep -r "spilo-14:2.1-p6" .
./manifests/configmap.yaml:  docker_image: registry.opensource.zalan.do/acid/spilo-14:2.1-p6
./manifests/operatorconfiguration.crd.yaml:                default: "registry.opensource.zalan.do/acid/spilo-14:2.1-p6"
./manifests/complete-postgres-manifest.yaml:  dockerImage: registry.opensource.zalan.do/acid/spilo-14:2.1-p6
./manifests/postgresql-operator-default-configuration.yaml:  docker_image: registry.opensource.zalan.do/acid/spilo-14:2.1-p6
./README.md:The newest Spilo image is: `registry.opensource.zalan.do/acid/spilo-14:2.1-p6`
./pkg/util/config/config.go:	DockerImage             string            `name:"docker_image" default:"registry.opensource.zalan.do/acid/spilo-14:2.1-p6"`
./pkg/controller/operator_config.go:	result.DockerImage = util.Coalesce(fromCRD.DockerImage, "registry.opensource.zalan.do/acid/spilo-14:2.1-p6")
./charts/postgres-operator/crds/operatorconfigurations.yaml:                default: "registry.opensource.zalan.do/acid/spilo-14:2.1-p6"
./charts/postgres-operator/values.yaml:  docker_image: registry.opensource.zalan.do/acid/spilo-14:2.1-p6
```

also #2078 should be marked as resolved.